### DISCUSSION TITLE
There is a problem when I use npx prisma init

### DISCUSSION THREAD

**Discussion by xxlbb123** [prisma/prisma#22673] [Q&A]
### Bug description

 Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/e95e739751f42d8ca026f6b910f5a2dc5adeaeee/windows/schema-engine.exe.sha256 failed, reason: read ECONNRESET

### Expected behavior

Smooth implementation

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
The URL works fine for me from locations around the world. Is something blocking access to this URL for you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
